core/integration/platform_test.go

```diff
@@ -111,10 +99,10 @@ func TestPlatformCrossCompile(t *testing.T) {
 		eg.Go(func() error {
 			ctr := c.Container().
 				From("crazymax/goxx:latest").
-				WithMountedCache(gomodCache, "/go/pkg/mod").
-				WithMountedCache(gobuildCache, "/root/.cache/go-build").
-				WithMountedDirectory("/src", thisRepo).
-				WithMountedDirectory("/out", "").
+				WithMountedCache(c.CacheVolume("gomod"), "/go/pkg/mod").
+				WithMountedCache(c.CacheVolume("gobuild"), "/root/.cache/go-build").
```

Comment:
Nice that you can inline these now especially, since they're often throwaway/single-use with names made up on-the-spot.